wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -1550,14 +1550,41 @@ private function execute_describe() {
 	 */
 	private function execute_update() {
 		$this->rewriter->consume(); // Update.
-
-		$params = array();
+		$limit    = $this->rewriter->peek(
```

Comment:
Technically this will also kick in here:

```sql
UPDATE table SET value = 1 WHERE value in (SELECT id from another_table);
```

This isn't a big issue, but since getting it exactly right would require a complex subquery analyzer, why don't we ditch the entire "add a custom SELECT conditionally" and just always transform the query to its `WHERE rowid in` form?

Reply:
In my mind, this sounds more complex than adding the SELECT statement conditionally, only to those two cases we now that they are not supported. Can you elaborate more?
Also, is it ok to consider this an enhancement and create another issue for that?